packages/databyss-ui/editor/state/page/actions.js

```diff
@@ -16,6 +16,7 @@ import {
   SHOW_FORMAT_MENU,
   SHOW_NEW_BLOCK_MENU,
   UPDATE_SOURCE,
+  DEQUEUE_NEW_SOURCES,
```

Comment:
The plural is misleading. I was imagining it taking an array, but the singular works too. Make the action name singular.